src/app/static/scripts/run-dependencies-for-integration-tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -ex
+
+HERE=$(readlink -f "$(dirname $0)")
+NETWORK=hint_nw
+HINT=hint
+$HERE/../../../../scripts/run-dependencies.sh
+HINT_VERSION=mrc-433_integrationtests
```

Comment:
This should be changed to master once this branch is merged. I think we want to pin the version like this (either hard-coded here or we could do it via a config file as we do for hintr) rather than get the current branch because otherwise we end up with a bit of an awkward dependency on Travis - the front-end build won't pass unless the back-end build has.